volttrontesting/platform/dbutils/test_mysqlfuncts.py

```diff
@@ -23,7 +23,7 @@
 
 
 IMAGES = [
-    "mysql:latest"
+    "mysql:latest",
```

Comment:
the latest tag with docker is a double edge sword...it doesn't mean what you think it means.  https://vsupalov.com/docker-latest-tag/

Specify the latest as 8.blah or whatever not the latest tag.

Reply:
Thanks for the article. Will change the tag to 8.0